migen/build/platforms/qm_xc6slx16_sdram.py

```diff
@@ -0,0 +1,57 @@
+from migen.build.generic_platform import *
+from migen.build.xilinx import XilinxPlatform
+
+
+_io = [
+    ("led1", 0, Pins("T9"), IOStandard("LVCMOS33"), Drive(24), Misc("SLEW=QUIETIO")),
+
+    ("led3", 0, Pins("R9"), IOStandard("LVCMOS33"), Drive(24), Misc("SLEW=QUIETIO")),
```

Comment:
Other platforms simply have "led" and use the resource number to differentiate them (``("led", 0, ... ("led", 1, ...``).

Is there a reason to do it this way and to number the led 1 and 3 instead of 0 and 1?

Reply:
Hello, I wanted the number to match to what's written on the board. I renamed the resource to user_led in latest commit.